solidity/app/cl_utils.js

```diff
@@ -0,0 +1,33 @@
+require('./cl_wallet.js')
+
+const Eth = require('ethjs')
+
+global.clUtils = global.clUtils || {
+  personalAccount: '0x9CA9d2D5E04012C9Ed24C0e513C9bfAa4A2dD77f',
+  toWei: function toWei (eth) {
+    return (parseInt(eth.toString()) * 10 ** 18).toString()
```

Comment:
Because we're often dealing with numbers in a variety of bases I feel like this should include the radix e.g. `parseInt(eth.toString(), 10)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt